liberty-archetype-mp/src/main/resources/archetype-resources/src/test/java/it/GreetingIT.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
```

Comment:
For some reason when I build the project locally the `GreetingIT` in `target` has a bunch of compilation errors:

```
[INFO] [INFO] -------------------------------------------------------------
[INFO] [ERROR] COMPILATION ERROR :
[INFO] [INFO] -------------------------------------------------------------
[INFO] [ERROR] /Users/ctianus.ibm.com/Desktop/ci.maven/liberty-archetype-mp/target/test-classes/projects/it/project/ServletSample/src/test/java/${groupId}/it/GreetingIT.java:[1,1] class, interface, or enum expected
[INFO] [ERROR] /Users/ctianus.ibm.com/Desktop/ci.maven/liberty-archetype-mp/target/test-classes/projects/it/project/ServletSample/src/test/java/${groupId}/it/GreetingIT.java:[1,2] class, interface, or enum expected
[INFO] [ERROR] /Users/ctianus.ibm.com/Desktop/ci.maven/liberty-archetype-mp/target/test-classes/projects/it/project/ServletSample/src/test/java/${groupId}/it/GreetingIT.java:[1,3] class, interface, or enum expected
...
```

The problem is that the `GreetingIT` in `target` is missing the first `/` at the beginning of the comment block. I'm not sure why this is happening because it's correct in this file. Can you see if this is also happening for you?

Reply:
I don't get this problem and so I'm not able to reproduce it.  :( Is there anything you could try to get around it, e.g. add a blank line at the start of the file? :/

Reply:
So you don't see the missing `/` in the `GreetingIT.java` in target? When do you a `clean install` on the generated archetype project does the compile succeed?

Reply:
No I don't see the problem.  The '/' is there and the generate code all builds fine and the tests pass.  Could you try my earlier suggestions?

Reply:
Sure. It might just be on my end though (seemed like the AppVeyor build worked fine).